tests/__init__.py

```diff
@@ -0,0 +1,19 @@
+"""We'd like to unit-test the code after merely running 'setup.py build' which
```

Comment:
I’m not going to add this: Just use `setup.py build_ext -i` and it’ll work. Or install cutadapt into a virtualenv in develop mode with `pip install -e .`. Or use `tox`, which will do that for you. I can document this.